pkg/quota/image/imagestreammapping_evaluator.go

```diff
@@ -27,7 +27,7 @@ func NewImageStreamMappingEvaluator(osClient osclient.Interface) kquota.Evaluato
 	matchesScopeFunc := func(kapi.ResourceQuotaScope, runtime.Object) bool { return true }
 
 	return quotautil.NewSharedContextEvaluator(
-		"ImageStreamMapping admission evaluator",
+		"Evaluator.ImageStreamMapping.Admission",
```

Comment:
Can we have this and `Evaluator.ImageStream.ResourceQuotaController` being constants?